std/uni.d

```diff
@@ -1898,23 +1898,29 @@ public alias CodepointSet = InversionList!GcPolicy;
 */
 public struct CodepointInterval
 {
-pure:
-    uint[2] _tuple;
-    alias _tuple this;
+    private uint[2] _tuple;
+    alias tuple this;
 
 @safe pure nothrow @nogc:
-
+    /// Constructor
     this(uint low, uint high)
     {
         _tuple[0] = low;
         _tuple[1] = high;
     }
+
+    /// Support for equality testing
     bool opEquals(T)(T val) const
```

Comment:
If the code is old enough, it might have needed an explicit `opEquals` to deal with `const`, but I don't think that that's the case anymore. You should probably verify that though. Another possibility is so that it can be compared with either a tuple or an array.